modules/ROOT/pages/database-internals/store-formats.adoc

```diff
@@ -0,0 +1,44 @@
+= Store formats
+:description: This page describes store formats in Neo4j.
+
+The store format is a core component of a database in Neo4j, describing how data is stored on disk and how to read it.
+
+== Store formats in Neo4j
+Neo4j has several defined store formats, each with a set of different features and limitations. The store format of a database is decided when the database is created, most commonly controlled by the `db.format` setting in the neo4j.conf file, or as an argument to the tool creating the database, e.g `import` or `copy` tools. The current available formats are
+
+* `aligned`
+* `standard`
+* `high_limit` label:enterprise-edition[]
+* `block` label:enterprise-edition[]
+
+== Properties of a store formats
+The store format is responsible for how data is written to disk and how to read it. Some key aspects that may differ between formats
+
+* The performance and resource consumption of read and write transactions. An operation may be faster on one store format than another.
+* The physical size on disk. Two databases with identical logical data on different format may have different size.
+* Performance in memory constrained environments. The same query on different formats may have different page cache hit ratio when the database does not fully fit in the page cache
```

Comment:
```suggestion
* Performance in memory-constrained environments.
The same query on different formats may have a different page cache hit ratio when the database does not entirely fit in the page cache.
```

Reply:
Can we add one more bullet here? Something like
> Which files/filenames exists in the database directory